util.sh

```diff
@@ -47,6 +47,28 @@ function build_plist() {
     fi
 }
 
+################################################################################
+# Build font instances from designspace and UFOs.
+# Arguments:
+#     Path to designspace source.
+#     Output formats, as separate strings.
+################################################################################
+function build_designspace() {
+    filename=$(basename "$1"); filename="${filename%.*}"
+    fontmake -m "$1" -o "${@:2}" --expand-features-to-instances
+    for format in "${@:2}" ; do cp master_$format/$filename-*.$format instance_$format/; done
+}
+
+
+################################################################################
+# Build variable font from designspace and UFOs.
+# Arguments:
+#     Path to designspace source.
+################################################################################
+function build_designspace_variable() {
+    fontmake -m "$1" -o variable --expand-features-to-instances
```

Comment:
```suggestion
    fontmake -m "$1" -o variable
```

the --expand-features-to-instances is only used when making static fonts, does not affect the variable output. I would actually expect fontmake to bark at you when you pass that option with variable output